File: Solutions/2433/2433.py
class Solution:
    def findArray(self, pref: List[int]) -> List[int]:
        pre=pref[0];
        n=len(pref)
        for i in range(1,n):
            pref[i]=pre^pref[i]
            pre^=pref[i]
        return 

File: Solutions/2433/2433.java
class Solution {
    public int[] findArray(int[] pref) {
        int pre=pref[0];
        for(int i=1;i<pref.length;++i){
            pref[i]=pre^pref[i];
            pre^=pref[i];
        }
        return pref;
    }
}